src/Features/Core/Portable/ConflictMarkerResolution/AbstractConflictMarkerCodeFixProvider.cs

```diff
@@ -220,10 +246,73 @@ private bool IsConflictMarker(SourceText text, SyntaxTrivia trivia, char ch)
                 text[trivia.SpanStart] == ch;
         }
 
+        private async Task<Document> FixAllAsync(
+            Document document, ImmutableArray<Diagnostic> diagnostics,
+            string equivalenceKey, CancellationToken cancellationToken)
```

Comment:
specialized fix-all since the default BatchFixAll provider gets completely thrown off by the types of edits this feature performs.  The well known batch-fix-all provider does a diff that gets completely screwy, causing the different fixes to look like they interfere witheach other.  

Here there is no diffing.  We just process all the independent sections and produce edits which can never conflict.